Cubical/Algebra/CommRing/Instances/IntMod.agda

```diff
@@ -0,0 +1,52 @@
+{-# OPTIONS --safe --experimental-lossy-unification #-}
+
+module Cubical.Algebra.CommRing.Instances.IntMod where
+
+open import Cubical.Foundations.Prelude
+
+open import Cubical.Algebra.CommRing
+open import Cubical.Algebra.Ring
+open import Cubical.Algebra.Group.Instances.IntMod
+open import Cubical.Algebra.Monoid.Base
+open import Cubical.Algebra.Semigroup.Base
+open import Cubical.Algebra.Group.Base
+open import Cubical.Algebra.AbGroup
+open import Cubical.Algebra.Semigroup
+
+open import Cubical.Data.Empty as ⊥
+open import Cubical.Data.Nat hiding (_+_)
+open import Cubical.Data.Fin
+open import Cubical.Data.Fin.Arithmetic
+
+open CommRingStr
+open IsCommRing
+open IsMonoid
+open IsSemigroup
+open IsRing
+open AbGroupStr
```

Comment:
Do you need this big of a header for this one result?

Reply:
Unfortunately... They lines become so incredibly long otherwise.

Reply:
What about all the imports?

Reply:
fixed

Reply:
Actually, no, sorry I need this verbose definition (with all the imports). If I use makeIsCommRing or something like that, it breaks too much stuff, which only seem to be fixable by entering transport hell. I think this is the best I can do.